admin/pre-reg/templates/edit_draft_registration.html

```diff
@@ -0,0 +1,103 @@
+{% extends 'base.html' %}
```

Comment:
We should consider putting this and similar files under admin/templates/prereg directory unless we foresee this app becoming a standalone from the admin module.